packages/astro/src/@types/astro.ts

```diff
@@ -2348,15 +2348,15 @@ export interface AstroUserConfig {
 		 *
 		 * 2. **Edit the collection definition**. Your updated collection requires a `loader`, and the option to select a collection `type` is no longer available.
 		 *
-		 *     ```diff
+		 *     ```ts ins={3,8} del={7}
 		 *     // src/content/config.ts
 		 *     import { defineCollection, z } from 'astro:content';
-		 *     + import { glob } from 'astro/loaders';
+		 *      import { glob } from 'astro/loaders';
```

Comment:
```suggestion
		 *     import { glob } from 'astro/loaders';
```
I'm not sure if the indentation is correct. It's a bit hard to see

Reply:
It is always so hard to see, thank you!